CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributor Guideline
+
+
+## Commit
+
+> One golden rule: follow [the commit format][commit-format]
+
+We use [standard-version][standard-version] to automatically generate [CHANGELOG.md][CHANGELOG.md] from the `git commit` history. Please **do** follow [the commit format][commit-format] before commiting.
+
+
+## PR
+
+Simple rules:
+
+* Do rebase before merging.
```

Comment:
Yes. We don't use GitHub merge button for collaborator created PRs. Instead, we do:

1. Rebase the branch to latest master so that it can be ff-merged.
2. Force pushed the branch to let GitHub know the branch is updated.
3. Merge the branch to master (should be fast-forward merge).
4. Push master to GitHub (origin). GitHub will automatically close the PR :)